packages/playground/src/samples/Sample.ts

```diff
@@ -0,0 +1,3 @@
+import { FormProps } from '@rjsf/core';
+
+export type Sample = Omit<FormProps, 'validator'>;
```

Comment:
Main cause for the playground example breaking was that it didn't get updated when the API changed in v5. This is more easily caught if the samples are all typed.